tools/export.py

```diff
@@ -223,31 +212,30 @@ def main(args):
         cfg.load_from = args.checkpoint
         cfg.resume_from = None
         compression_ctrl, model = wrap_nncf_model(model, cfg, None, get_fake_input)
-        compression_ctrl.prepare_for_export()
+        # TODO: apply the following string for NNCF 1.5.*
+        # compression_ctrl.prepare_for_export()
     # END nncf part
 
-    with_text = False
-    if args.target == 'openvino' and not args.alt_ssd_export:
-        if hasattr(model, 'roi_head'):
-            stub_roi_feature_extractor(model.roi_head, 'bbox_roi_extractor')
-            stub_roi_feature_extractor(model.roi_head, 'mask_roi_extractor')
-            if getattr(model.roi_head, 'with_text', False):
-                with_text = True
-                stub_roi_feature_extractor(model.roi_head, 'text_roi_extractor')
-
     mmcv.mkdir_or_exist(osp.abspath(args.output_dir))
     onnx_model_path = osp.join(args.output_dir,
                                osp.splitext(osp.basename(args.config))[0] + '.onnx')
 
     with torch.no_grad():
         export_to_onnx(model, fake_data, export_name=onnx_model_path, opset=args.opset,
                        alt_ssd_export=getattr(args, 'alt_ssd_export', False),
-                       verbose=True)
+                       target=args.target, verbose=True)
         add_node_names(onnx_model_path)
         print(f'ONNX model has been saved to "{onnx_model_path}"')
 
     optimize_onnx_graph(onnx_model_path)
 
+    # ?
+    with_text = False
+    if args.target == 'openvino' and not args.alt_ssd_export:
+        if hasattr(model, 'roi_head'):
+            if getattr(model.roi_head, 'with_text', False):
+                with_text = True
+    
```

Comment:
?

Reply:
@Ilya-Krylov Old-style tracer stubs have been gone, all custom exporting logic is handled in a unified way now. Please give us a hint on where do you think is a better place for this code? Should we move it to `export_to_onnx` function?

Reply:
let think about it a bit later, leave it here for a while. I've tested on text-spotting model and it works.